minigalaxy/installer.py

```diff
@@ -19,10 +19,10 @@ def install_game(game, installer):
         error_message = move_and_overwrite(game, tmp_dir, game.install_dir)
     if not error_message:
         error_message = copy_thumbnail(game)
-    if not error_message:
-        error_message = remove_installer(installer)
-    else:
+
+    if not Config.get("keep_installers"):
```

Comment:
I don't like it. Convention:
```
if not error_message:
    error_message = action()
```
Allowed us to obtain error_message from action and report it for user in error message window. We are now cutting of that possibility from remove_installer(). Checking Config.get("keep_installer") can be done inside of remove_installer().

Reply:
Perhaps, but we do need to make sure the installer is deleted if the verification fails. Do you think it would be better for verify_installer_integrity to delete the installer by itself when there is an issue?

Reply:
I just re-read the old code and I understand now. I'll restore it and update remove_installer